### PR TITLE
Feat: Cli improvements

### LINKED ISSUE #469: Extend Robyn cli
With the release of v0.28.0 - robyn has a `create` function.

Extend it to add

1. Docker config
2. nginx/caddy config
3. suggest more ideas.

### PR DESCRIPTION
**Pull request by Oluwaseun241**
**Description**
This is an improvement to the CLI adding:
-Docker config
-Nginx config

This PR fixes #469

### CODE REVIEW

**Review by sansyrox — COMMENTED** [review]
Hey @Oluwaseun241 👋 

Thank you for the PR . Great work! 😄 I am traveling atm but I have a few comments.

**Review by sansyrox — COMMENTED** [review]
@Oluwaseun241 , thanks for the PR :D

I have a few suggestions

File: robyn/__main__.py [hunk 1]
```diff
@@ -6,6 +6,7 @@
 
 def create():
     project_dir = input("Enter the name of the project directory: ")
+    docker = input("Do you want Docker? ")
```
Comment (sansyrox) [inline robyn/__main__.py:9]: Can we change the input prompt to ask for something like

```
project_dir = input("Need Docker? (Y/N)")
```

File: robyn/__main__.py [hunk 2]
```diff
@@ -33,6 +34,39 @@ def index():
             """
         )
 
+    # DockerFile configuration
+    if docker == "yes":
```
Comment (sansyrox) [inline robyn/__main__.py:38]: create a sanitization function here, which will take in the value of the variable and check for the `Y/N` value

File: robyn/__main__.py [hunk 3]
```diff
@@ -33,6 +34,39 @@ def index():
...
+app.py", "--log-level=DEBUG"]
+
+                """
+            )
+    elif docker == "no":
+        print("Docker not included")
+    else:
+        print("Unknown Command")
+
+    # Final
```
Comment (sansyrox) [inline robyn/__main__.py:69]: we can get rid of this comment?

File: robyn/__main__.py (hunk 1 above)
Comment (sansyrox) [inline robyn/__main__.py:9]: Can we change the input prompt to ask for something like

```
project_dir = input("Need Docker? (Y/N)")
```

File: robyn/__main__.py [hunk 4]
```diff
@@ -6,7 +6,16 @@
 
 def create():
     project_dir = input("Enter the name of the project directory: ")
+    docker = input("Need Docker? (Y/N) ")
+
     # Initailize a new Robyn project
+    def check(value):
+        while value not in ["Y", "N"]:
```
Comment (sansyrox) [inline robyn/__main__.py:13]: @Oluwaseun241 , I don't think I want to a loop here. Can you add an `if` check to see if the value is not equal to `Y` or `y`, we replace it with `N`

Comment (Oluwaseun241) [reply robyn/__main__.py:13]: The loop was there because I don't want them to restart the CLI when there is a mistakenly or genuinely unknown command

**Review by sansyrox — APPROVED** [review]
Thank you @Oluwaseun241 😄 

Great work 🔥

**Pull request merged**